src/app/category/category.component.css

```
@@ -0,0 +1,12 @@
+.category-name {
+  text-align: left;
+  padding: 0.2em 0 0.2em 0.2em;
+}
+
+.category-content{
+  margin-left: 1em;
+}
+
+.category-name:hover {
+  background-color:rgb(230, 230, 230);
+}
```

```suggestion
}

```

All files should end with a newline. Normally, we'd configure a formatter to handle this for us automatically.

For now, please make the adjustment and keep this in mind.